tools/rna_tools/targetfinder/TargetFinder_threads.xml

```diff
@@ -0,0 +1,57 @@
+<tool id="targetfinder_threads" name="targetfinder_threads" version="0.1.0">
```

Comment:
That is because of what is provided by targetfinder package. 

the major difference is the input - targetfinder only allows a string of sequence while targetfinder_threads takes a fasta file.